packages/suite/src/views/dashboard/components/T2B1PromoBanner.tsx

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import styled from 'styled-components';
+import { rgba } from 'polished';
+
+import { EventType, analytics } from '@trezor/suite-analytics';
+import { TREZOR_URL } from '@trezor/urls';
+
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { selectT2B1PromoBanner } from 'src/reducers/suite/suiteReducer';
+import { Button, Image, variables } from '@trezor/components';
+import { setFlag } from 'src/actions/suite/suiteActions';
+import { Translation, TrezorLink } from 'src/components/suite';
+
+// banner size 800px
+const SMALL_SCREEN_BREAKPOINT = 832;
+// banner size 640px
+const MOBILE_BREAKPOINT = 671;
+
+const BannerWrapper = styled.div`
+    display: grid;
+    grid-template-columns: 338px 1fr minmax(109px, 145px) 42px;
+    grid-template-rows: 42px 1fr 0;
+    column-gap: 8px;
+    background-color: #0f6148;
+    height: 167px;
+    width: 100%;
+    border-radius: 12px;
+    margin-top: 16px;
+    margin-bottom: -18px;
+    padding-left: 24px;
+    padding-right: 24px;
+
+    @media screen and (max-width: ${SMALL_SCREEN_BREAKPOINT}px) {
+        height: 132px;
+        padding-left: 12px;
+        padding-right: 12px;
+    }
+
+    @media screen and (max-width: ${MOBILE_BREAKPOINT}px) {
+        height: 190px;
+        grid-template-columns: 171px 1fr 0 32px;
+        grid-template-rows: 36px 1fr 36px;
+    }
+`;
+
+const TrezorSafe3Logo = styled(Image)`
+    grid-column: 1;
+    grid-row: 1;
+    height: fit-content;
+    filter: none;
+
+    padding-top: 28px;
+
+    @media screen and (max-width: ${SMALL_SCREEN_BREAKPOINT}px) {
+        padding-top: 16px;
+    }
+`;
+
+const NextGenerationText = styled.p`
+    grid-column: 1;
+    grid-row: 2;
+
+    margin-bottom: 24px;
+    align-self: end;
+
+    position: relative;
+    font-size: 32px;
+    letter-spacing: -0.65px;
+    line-height: 34.6px;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    z-index: 1;
+
+    @media screen and (max-width: ${SMALL_SCREEN_BREAKPOINT}px) {
+        font-size: 24px;
+        line-height: 26px;
+        margin-bottom: 16px;
+    }
+
+    @media screen and (max-width: ${MOBILE_BREAKPOINT}px) {
+        align-self: end;
+        margin-bottom: 18px;
+    }
+`;
+
+const NextGenerationTextBlock = styled.span`
+    color: ${({ theme }) => theme.TYPE_WHITE};
+`;
+
+const NextGenerationTextBlockHighlight = styled(NextGenerationTextBlock)`
+    color: #9be887;
+    text-wrap: nowrap;
```

Comment:
I also wanna write `text-wrap` sometimes :DD

```suggestion
    white-space: nowrap;
```

Reply:
But it works! Yea I always mix it up

Reply:
8a9038a9c5f8f95112ec69ae8911c6b9561098fa

Reply:
I actually didn't know! :D I got a warning from stylelint. And I believe that the [browser support](https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap#browser_compatibility) is not good.